Assets/Scripts/Faction.cs

```diff
@@ -0,0 +1,164 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[System.Serializable]
+public class Faction
+{
+    /// <summary>
+    /// name of the faction
+    /// </summary>
+    [SerializeField] private string factionName; //make this field adjustable in the inspector
```

Comment:
Move this down to the other public string FactionName on LINE 19 CAUSE IT'S CONFUSING HERE BEfore the comment

Reply:
Done